### ISSUE TITLE
Unable to load Colaboratory.

### ISSUE THREAD

**Issue by pursh2002** [googlecolab/colabtools#485]
Unable to load Colaboratory.

Insufficient permissions for the specified parent.
lb@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:544:302
py@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:1348:360
QG@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:1567:360
HI/<@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:1655:331
Ca@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:509:334
Aa.prototype.throw_@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:508:402
Ea/this.throw@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:510:246
c@https://colab.research.google.com/v2/external/external_polymer_binary.js?vrz=colab-20190314-085300-RC00_238451813:519:72

**Comment by blois** [comment]
Is this perhaps a Drive notebook stored in Team Drive?

**Comment by pursh2002** [comment]
The same problem exists with the new notebook

**Comment by YaminiGoel** [comment]
@pursh2002 Did your problem get resolved? I am facing the same issue...

**Comment by blois** [comment]
I'd be curious if you can create a Colab notebook from the Drive UI:

![image](https://user-images.githubusercontent.com/2152569/62306981-a7078680-b437-11e9-8d5c-c30cadeb0b08.png)

**Comment by chanil1218** [comment]
I have the same issue when I am trying to copy public Colab notebook to Google Drive.
I resolved this error with removing **Colab Notebooks** folder that is shared with me.
(I had two Colab Notebooks directory in my Google Drive. One that I used to use and other shared with me.)

Long story to short, I think **the specified parent** in error message points implicitly(sadly not explicitly) **Colab Notebooks**.

https://support.google.com/drive/thread/12526192?msgid=14922479
